mne/decoding/transformer.py

```diff
@@ -286,7 +296,7 @@ def __init__(self, sfreq=2 * np.pi, fmin=0, fmax=np.inf, bandwidth=None,
         self.verbose = verbose
         self.normalization = normalization
 
-    def fit(self, epochs_data, y):
+    def fit(self, epochs_data, y=None):
```

Comment:
hmm ... I don't think it's a bug. You need to give `y` in supervised learning. I'd revert this change ...